### PR TITLE
WIP: Working on helpers

### PR DESCRIPTION
**Pull request by jisson**

### CODE REVIEW

File: djhubspot/helpers.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
+import logging
+from _decimal import InvalidOperation
```
Comment (metamatik) [inline djhubspot/helpers.py:2]: `_decimal` looks like a typo ;)

File: djhubspot/helpers.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+from django.conf import settings
+
+from hubspot3.associations import AssociationsClient
+from hubspot3.deals import DealsClient
+from hubspot3.products import ProductsClient
+from money.money import Money
```
Comment (metamatik) [inline djhubspot/helpers.py]: I haven't heard of `money` yet, I'm going to have a look — sounds interesting.

**Review by metamatik — COMMENTED** [review]
LGTM so far :)

File: djhubspot/helpers.py (hunk 2 above)
Comment (jisson) [reply djhubspot/helpers.py]: I've seen it in that [article (sametmax)](http://sametmax.com/stack-python-en-2019/). Actually i am currently using https://github.com/vimeo/py-money as I was not sure that [money](https://github.com/carlospalol/money) totally support python 3.7.

**Pull request merged**